requirements/requirements_doc.txt

```diff
@@ -1,4 +1,4 @@
-Sphinx>=4.4
-numpydoc>=1.2
-ansys_sphinx_theme>=0.4.0
-Sphinx-copybutton>=0.4
+Sphinx==5.0.2
+numpydoc==1.4.0
+ansys-sphinx-theme==0.4.2
+sphinx-copybutton==0.5
```

Comment:
I know we are familiar with this (i.e. dependabot only updates reqs in case of "==", since => is too flexible). But, we should probably added to the pyansys/dev-guide. I'll open an issue in case it is not there yet.

Reply:
Exactly, I just noticed that. Could you please link this new issue to this PR? Just make a short reference so we can track it.